actioncable/lib/action_cable/subscription_adapter/postgresql.rb

```diff
@@ -12,16 +12,16 @@ def initialize(*)
 
       def broadcast(channel, payload)
         with_connection do |pg_conn|
-          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel)}, '#{pg_conn.escape_string(payload)}'")
+          pg_conn.exec("NOTIFY #{pg_conn.escape_identifier(channel.to_s)}, '#{pg_conn.escape_string(payload)}'")
```

Comment:
What about using the implicit conversion, which will allow for string-like objects, while failing (loudly) for non-string-like things?

```ruby
String(channel)
```